src/electron/process.ts

```diff
@@ -22,89 +23,107 @@ import * as path from 'path';
 //       (which may be immediately after calling #startInternal if, e.g. the binary cannot be
 //       found).
 export class ChildProcessHelper {
-  private process?: ChildProcess;
-  protected isInDebugMode = false;
+  private readonly processName: string;
+  private subProcess?: ChildProcess = null;
```

Comment:
For consistency of names:
```suggestion
  private childProcess?: ChildProcess = null;
```

At first I thought you were starting yet another process.